### DISCUSSION TITLE
Backup and autosave in Github site editor: is it missing?

### DISCUSSION THREAD

**Discussion by BYTESHAUS** [community/community#110781] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It's usual to write markdown documents in GitHub just in a browser (to get the Preview, unavailable in other tools like Visual Studio).
 You may not want to submit it ASAP but keep it open and edit.
On browser crash, restart, or occasional close you may either lose your work or need to backup somewhere.
An option such as auto-recovery, or save as draft would be great but i haven't found any.

**Comment by GeigerJ2** [comment]
Commenting here to (possibly) keep updated, should something change about that. I cannot believe such a basic feature as auto-saving on text fields is missing...

**Reply by BYTESHAUS** [reply]
What makes matters worse, changes may be sporadically lost without a prompt after hibernation. 
Meanwhile, Ctrl+C with Win+V is our friend.

**Reply by Chealer** [reply]
@BYTESHAUS wrote:
> What makes matters worse, changes may be sporadically lost without a prompt after hibernation.

That is probably because Windows crashes suspending, or because of forced automatic Windows updates. If the latter, see:
1. [Disable automatic restarts of Windows 11 (Pro)](https://superuser.com/questions/1817039/disable-automatic-restarts-of-windows-11-pro)
2. [How to *disable* automatic reboots in Windows 10?](https://superuser.com/questions/957267/how-to-disable-automatic-reboots-in-windows-10)
3. On Home edition: [Windows Starter Edition, Home and Home Premium do not include gpedit, how do I install it?](https://superuser.com/questions/1018145/windows-starter-edition-home-and-home-premium-do-not-include-gpedit-how-do-i-i)

> Meanwhile, Ctrl+C with Win+V is our friend.

Windows clipboard history or any clipboard manager (like Ditto or CopyQ) are indeed some of the best workarounds, but it requires discipline. The problem is Windows reboots at night, and the moment you would need to save is the one you are least alert: the time when you go to bed. I lost at least 3 reports/comments due to automatic restarts this year, even though I use multiple clipboard managers and have been aware of this issue since September.

[🅭🄍](https://www.philippecloutier.com/Common+infrastructure+licensing#forum)

**Comment by Chealer** [comment]
[There is reportedly something for draft merge requests](https://github.com/orgs/community/discussions/68448#discussioncomment-13743920), and [it is complicated](https://github.com/orgs/community/discussions/68448#discussioncomment-14378808). But you are unfortunately right that it is missing and/or broken. That discussion treats comments on merge requests, and #42581 covers ticket creation. I found nothing about commenting tickets or discussions.

The previous discussions mention a couple hacks which may prevent this, but I did not try any. Preventing Windows from spurious reboots seems less complicated, but still surprisingly hard in particular if you are using the Home edition (I just spent 4 hours on trying something, still need to confirm it works). And yeah, that won't cover browser crashes, but the worst combination seems to be GitHub+Windows, although Edge **might** reduce losses.
